### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by initseis**
In this branch we:

- Added a navbar
- Added a footer
- Create event listeners for links to display and hide sections
- Add style
- Add date time

### CODE REVIEW

File: scripts/main.js [hunk 1]
```diff
@@ -2,6 +2,14 @@ const mainForm = document.getElementById('main-form');
...
 const bookAuthor = document.getElementById('book-author');
 const bookList = document.querySelector('.book-list');
+const listA = document.getElementById('list-option');
+const newBookA = document.getElementById('new-book-option');
+const contactA = document.getElementById('contact-option');
+const logoA = document.getElementById('awesome-logo');
+const dateTime = document.getElementById('date-time');
+const d = new Date();
+const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+dateTime.innerHTML = `${months[d.getMonth()]} ${d.getDate()}th  ${d.getUTCFullYear()}, ${d.getHours()}:${d.getMinutes()}:${d.getSeconds()}`;
```
Comment (mimipeshy) [inline scripts/main.js]: - [ ] From the project specifications, you were supposed to use the [luxon ](https://moment.github.io/luxon/#/install) library to [get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time). Kindly work on this as it a hard requirement before your project can be approved

**Review by mimipeshy — CHANGES_REQUESTED** [review]
## Hi @sevinchek and @abmAsadullah 👋🏾 👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
- Great git workflow ✔️ 
- Great PR summary ✔️ 
- Your UI looks awesome ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
------

**Review by abruzy — APPROVED** [review]
```markdown
Hi @sevinchek & @abmAsadullah 👋🏽, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by initseis** [comment]
Thanks, @abruzy.

**Pull request merged**